packages/gitmoji-changelog-core/src/utils.js

```diff
@@ -0,0 +1,32 @@
+const levenshtein = require('fast-levenshtein')
+
+function groupSentencesByDistance(texts = []) {
+  const textsWithSortedWords = texts.map(text => text.split(' ').sort().join(''))
+
+  const alreadyProcessedWords = new Set()
+  const keyGroups = new Map()
+
+  for (let i = 0; i < textsWithSortedWords.length; i += 1) {
```

Comment:
I'm not fond of i and j could we add more meaning to these variables ?

Reply:
i == indexesFromStart
j == indexesFromCurrent 

I don't know how to name them better

Reply:
I find it better it reflects more what are these indexes purpose 👍